File: app/src/main/java/com/zillennium/utswap/screens/setting/kycScreen/idTypeScreen/NationalIDFragment.java
package com.zillennium.utswap.screens.setting.kycScreen.idTypeScreen;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.RelativeLayout;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.zillennium.utswap.R;
import com.zillennium.utswap.databinding.FragmentKycNationalIdBinding;
import com.zillennium.utswap.screens.setting.kycScreen.CameraActivity;

public class NationalIDFragment extends Fragment {
    FragmentKycNationalIdBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = FragmentKycNationalIdBinding.inflate(getLayoutInflater());
        View root = binding.getRoot();

        RelativeLayout btnCameraFront = binding.btnCameraFront;
        btnCameraFront.setOnClickListener(view -> {
            Intent intent = new Intent(getContext(), CameraActivity.class);
            intent.putExtra("sample", R.drawable.ic_national_id_front);
            startActivity(intent);
        });

        RelativeLayout btnCameraBack = binding.btnCameraBack;
        btnCameraBack.setOnClickListener(view -> {
            Intent intent = new Intent(getContext(), CameraActivity.class);
            intent.putExtra("sample", R.drawable.ic_national_id_back);
            startActivity(intent);
        });

        return root;
    }



    @Override
    public void onResume() {
        super.onResume();
        binding.getRoot().requestLayout();
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/kyc/kycFragment/idTypeScreen/fragment/nationalID/NationalIDFragment.kt
package com.zillennium.utswap.screens.kyc.kycFragment.idTypeScreen.fragment.nationalID

import android.content.Intent
import androidx.navigation.fragment.findNavController
import com.zillennium.utswap.R
import com.zillennium.utswap.UTSwapApp
import com.zillennium.utswap.bases.mvp.BaseMvpFragment
import com.zillennium.utswap.databinding.FragmentKycNationalIdBinding
import com.zillennium.utswap.screens.kyc.kycFragment.idTypeScreen.camera.idCardCameraFragment.IDCardCameraFragment

class NationalIDFragment :
    BaseMvpFragment<NationalIDView.View, NationalIDView.Presenter, FragmentKycNationalIdBinding>(),
    NationalIDView.View {

    override var mPresenter: NationalIDView.Presenter = NationalIDPresenter()
    override val layoutResource: Int = R.layout.fragment_kyc_national_id

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                btnCameraFront.setOnClickListener {
                    findNavController().navigate(R.id.action_to_id_card_camera_kyc_fragment)
//                    val intent = Intent(UTSwapApp.instance, IDCardCameraFragment::class.java)
//                    intent.putExtra("sample", R.drawable.ic_national_id_front)
//                    startActivity(intent)
                }

                btnCameraBack.setOnClickListener {
                    findNavController().navigate(R.id.action_to_id_card_camera_kyc_fragment)
//                    val intent = Intent(UTSwapApp.instance, IDCardCameraFragment::class.java)
//                    intent.putExtra("sample", R.drawable.ic_national_id_back)
//                    startActivity(intent)
                }

            }
        } catch (error: Exception) {
            // Must be safe
        }
    }
}